client/src/blog/post.tsx

```diff
@@ -145,6 +179,7 @@ export function BlogPost(props: HydrationData) {
           <h1>{doc?.title}</h1>
           <AuthorDateReadTime metadata={blogMeta} />
           <RenderDocumentBody doc={doc} />
+          <PreviousNext metadata={blogMeta} />
```

Comment:
🟢 I wonder if we shouldn't pass in only what's needed? If we wrap `{ previous, next }` in a `links` object, we could pass in only that.